Docs/source/dataanalysis/advanced.rst

```diff
@@ -12,28 +12,25 @@ fine and auxiliary, see :ref:`the theory <theory>` for more details), and it is
 them (regular output return the auxiliary grid only). This page provides information to read
 raw data of all grids.
 
-Dump additional data
+Dump Additional Data
 --------------------
 
 In order to dump additional data in WarpX (mostly for debugging purpose), run the simulation
 with parameters (``warpx.plot_rho`` only when using back-transformed diagnostics)
 
 .. code-block:: python
 
-    warpx.plot_raw_fields = 1
```

Comment:
Now that you are mentioning this, I realize all the parameters here are obsolete.

Would it make sense to:
- Remove references to `plot_rho` and `plot_dive` for now?
- Replace `warpx.plot_raw_fields`, `warpx.plot_finepatch`, `warpx.plot_crsepatch` with `<diag_name>.plot_raw_fields`, `<diag_name>.plot_finepatch`, `<diag_name>.plot_crsepatch`

Reply:
This is a bit orthogonal to the original intent of this PR. But I think it is fine to still do the change in that same PR.